tests/tools/test_msc.lua

```diff
@@ -237,11 +238,11 @@
 	end
 
 	function suite.cflags_onSpecificWarnings()
-		enablewarnings { "enable" }
-		disablewarnings { "disable" }
-		fatalwarnings { "fatal" }
+		enablewarnings { "1234" }
+		disablewarnings { "5678" }
+		fatalwarnings { "9753" }
```

Comment:
Why is this being changed?

Reply:
At least to my understanding, the [MSC docs](https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-160) state the warnings are identified by a four-digit number.

Reply:
Undid those change.